chess/build.gradle

```diff
@@ -52,14 +52,61 @@ subprojects {
         outputs.dir dst
     }
 
-    genStubs {
+    task compileGenEngineStubs(type: JavaCompile) {
         dependsOn genEngineStubs
+        source = genEngineStubs.outputs
+        classpath = sourceSets.main.compileClasspath
+        destinationDir = sourceSets.main.output.classesDir
+        options.incremental = true
+        outputs.dir destinationDir
+    }
+
+    genStubs {
+        dependsOn compileGenEngineStubs
```

Comment:
I understand that we have stubs to be generated for classes in 2 different packages for this particular app. I think, stub generation phase shouldn't be doing compilation for each time we generate a stub in a package. We should have final compilation step after generating all the app stubs in the necessary packages. You may consider it in next PR.

Reply:
This will be handled in a separate PR.